apps/remote_control/lib/lexical/remote_control/code_intelligence/rename/module.ex

```diff
@@ -0,0 +1,164 @@
+defmodule Lexical.RemoteControl.CodeIntelligence.Rename.Module do
```

Comment:
I think you make a fair point, and I need to give more thought to the ergonomics aspect.

Additionally, a part that this PR is missing is that it currently only allows for renaming the module directly **under the cursor** (between two points, or between a point and a space). If we could support renaming the entire module completely, it might be more effective, though this could also be more challenging.